src/Meilisearch/TaskInfo.cs

```diff
@@ -0,0 +1,35 @@
+namespace Meilisearch
+{
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Update Status of the actions done.
+    /// </summary>
+    public class TaskInfo
+    {
+        /// <summary>
+        /// Gets or sets Uid for the task.
+        /// </summary>
+        public int Uid { get; set; }
```

Comment:
This comment could be applied to all of the properties. 

These `get; set;` are always required? So there are some of these properties that would not be necessary for example, only `Status` requires `set` but not `Uid`, `Type`. The general idea is to only use `get` and `set` when is really needed.

Reply:
To be honest, I tried to remove what I considered as the useless `set` in these classes, but it ended up with most of the tests failing, so I put back the `set` 😂 I know this is not the good way of coding, but I did not have the time to investigate 😢 But I would love knowing why and how to remove them 
Can we open an issue?

Reply:
Thanks @juchom!! :)
Should be fixed in another PR